.github/workflows/tests.yml

```diff
@@ -120,6 +120,9 @@ jobs:
     - name: Run brew audit --skip-style on all taps
       run: brew audit --skip-style
 
+    # TODO: remove --quiet when possible.
```

Comment:
@vidusheeamoli I added this so we can ensure that PRs like https://github.com/Homebrew/brew/pull/8505 are tested properly. Could you open a new PR to remove `--quiet` from here and fix and/or hide the current failures so that `brew typecheck` shows as clear by default? Thanks!

Reply:
On it! 😄